troubleshooting/troubleshoot-install.md

```diff
@@ -296,13 +296,28 @@ Error: INSTALLATION FAILED: unable to build kubernetes objects from release mani
 
 To disable PSP in your deployment:
 
+1. Backup your helm values with `helm get values kubecost > kubecost-values.yaml`
+2. Open `kubecost-values.yaml` and delete any references to `podSecurityPolicy` or `psp`
+3. Delete all helm secrets in the Kubecost namespace:
+
+    {% code overflow="wrap" %}
+    ```bash
+    # Get the list of secrets
+    kubectl get secrets -n kubecost
+    # Backup secrets to a file
+    kubectl get secrets -n kubecost -o yaml > kubecost-secrets.yaml
```

Comment:
@jessegoodier does the customer then need to recreate all the secrets required to re-install kubecost?  e.g. etlbackup secret, thanos secret, federated etl secret, cloud integration secret.

Reply:
no, the `helm get values` will backup all settings.

Reply:
step 1 above.